tests/components/octoprint/test_binary_sensor.py

```diff
@@ -0,0 +1,97 @@
+"""The tests for Octoptint binary sensor module."""
+import logging
+
+from pyoctoprintapi import OctoprintPrinterInfo
+
+from homeassistant.components.octoprint import binary_sensor as sensor
+from homeassistant.const import STATE_OFF, STATE_ON
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def test_OctoPrintPrintingBinarySensor_properties(hass):
+    """Test the properties."""
+    coordinator = DataUpdateCoordinator(hass, _LOGGER, name="octoprint-test")
+    coordinator.data = {"printer": {}}
+    test_sensor = sensor.OctoPrintPrintingBinarySensor(coordinator, "OctoPrint")
```

Comment:
Set up the integration and let it add an entity. Then assert the entity state via the core state machine `hass.states`.